charters/PORTABILITY/charter.md

```diff
@@ -0,0 +1,57 @@
+# Pipeline Portability Service
```

Comment:
It seems like the directory should be named _PipelinePortabilityService_ or _PIpelinePortability_ rather than _PORTABILITY_?

Reply:
Updated.